src/main/java/com/nike/cerberus/operation/composite/CreateEnvironmentOperation.java

```diff
@@ -21,16 +21,7 @@
 import com.nike.cerberus.command.cms.CreateCmsCmkCommand;
 import com.nike.cerberus.command.cms.CreateCmsConfigCommand;
 import com.nike.cerberus.command.composite.CreateEnvironmentCommand;
-import com.nike.cerberus.command.core.CreateBaseCommand;
-import com.nike.cerberus.command.core.CreateDatabaseCommand;
-import com.nike.cerberus.command.core.CreateEdgeDomainRecordCommand;
-import com.nike.cerberus.command.core.CreateLoadBalancerCommand;
-import com.nike.cerberus.command.core.CreateRoute53Command;
-import com.nike.cerberus.command.core.CreateSecurityGroupsCommand;
-import com.nike.cerberus.command.core.CreateVpcCommand;
-import com.nike.cerberus.command.core.CreateWafCommand;
-import com.nike.cerberus.command.core.GenerateCertificateFilesCommand;
-import com.nike.cerberus.command.core.UploadCertificateFilesCommand;
+import com.nike.cerberus.command.core.*;
```

Comment:
nit: I think we want to expand these as a part of our formatting.

Reply:
- [ ] expand